plugins/flytekit-memray/flytekitplugins/memray/__init__.py

```diff
@@ -0,0 +1,15 @@
+"""
+.. currentmodule:: flytekitplugins.wandb
+
+This package contains things that are useful when extending Flytekit.
+
+.. autosummary::
+   :template: custom.rst
+   :toctree: generated/
+
+   wandb_init
```

Comment:
replace mentions to wandb

Reply:
What exactly is supposed to be in this comment section of `__init__.py`?
I planned to just get rid of it actually?

Reply:
This is exposed as the landing page for the plugin in the docs, e.g. https://docs.flyte.org/en/latest/api/flytekit/plugins/wandb.html

Reply:
Good catch - updated!

Reply:
Removed adding docs from this PR, since we need a published version of thi plugin on pypi to continue with the process of adding docs to flytesnacks & flyte. Will follow up with docs as soon this is merged. 
Ok with you @eapolinario & can you please have a last review if times allows